src/main/java/frc/robot/subsystems/ShootSubsystem.java

```diff
@@ -42,24 +43,30 @@ public ShootSubsystem() {
     fly1.getPIDController().setFF(Constants.ShooterConstants.kFF.get());
 
     fly1.setOpenLoopRampRate(Constants.ShooterConstants.rampRate.get());
+
+    fly1.getPIDController().setOutputRange(0, 1);
   }
 
   public void setTargetRPM(double targetRPM) {
     // stuff :)
     // RPM = targetRPM;
     // SmartDashboard.putNumber("ShooterSetpoint", targetRPM);
     // if (flywheelMode == FlywheelControl.PID) {
-    //   fly1.getPIDController().setReference(targetRPM, ControlType.kVelocity);
+    fly1.getPIDController().setReference(targetRPM, ControlType.kVelocity);
     // } else if (flywheelMode == FlywheelControl.BANG_BANG) {
-    //   bangbang.setSetpoint(targetRPM);
+    // bangbang.setSetpoint(targetRPM);
     // }
 
-    fly1.set(targetRPM);
+    rpmSetpoint = targetRPM;
+    // bangbang.setSetpoint(targetRPM);
+    SmartDashboard.putNumber("rpm setpoint", targetRPM);
+
+    // fly1.set(targetRPM);
   }
 
   public boolean closeEnough() {
     // if (Math.abs(RPM - fly1.getEncoder().getVelocity()) > 50) {
-    //   return false;
+    // return false;
     // } else return true;
```

Comment:
this was commented out anyways but its very scuffed because i made it on the fly and also probably obsolete since our flywheel tuning is good; probably best to delete

Reply:
we should still use it for commands that need to wait until the flywheel is up to the proper speed